cniconf/watcher.go

```diff
@@ -125,3 +127,29 @@ func (w *watcher) apply(network metadata.Network) error {
 
 	return lastErr
 }
+
+func UpdateCNIConfigByHost(config interface{}, host metadata.Host) interface{} {
```

Comment:
Giving this function a generic name would be helpful if we introduce more keywords in future.
Say: `UpdateCNIConfigByKeywords`?